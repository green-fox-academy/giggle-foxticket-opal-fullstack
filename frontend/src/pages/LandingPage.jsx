import React from 'react';
import MainFeatures from '../components/MainFeatures/MainFeatures';

const LandingPage = () => {
<<<<<<< HEAD
  return <></>;
=======
  return (
    <>
      I am the landing page!
      <MainFeatures />
    </>
  );
>>>>>>> master
};

export default LandingPage;
