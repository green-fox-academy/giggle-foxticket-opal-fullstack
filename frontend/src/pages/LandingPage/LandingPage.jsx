import React from 'react';
import { Link } from 'react-router-dom';
import MainFeatures from '../../components/MainFeatures/MainFeatures';
import ImageComposition from '../../components/ImageComposition/ImageComposition';
import Footer from '../../components/Footer/Footer';
import VideoPlayer from '../../components/VideoPlayer/VideoPlayer';
import Button from '../../components/Button/Button';
import './LandingPage.styles.sass';
import SubscribeMail from '../../components/SubscribeMail/SubscribeMail';

const LandingPage = () => {
  return (
    <>
      <ImageComposition />
      <MainFeatures />
      <Link to="/shop">
        <Button buttonStyle="btn--danger--solid">Buy Tickets</Button>
      </Link>
      <VideoPlayer />
      <SubscribeMail />
      <Footer />
    </>
  );
};

export default LandingPage;
