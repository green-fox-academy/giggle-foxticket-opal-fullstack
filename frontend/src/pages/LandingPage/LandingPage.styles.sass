.btn--danger--solid
  position: relative
  left: 50%
  transform: translateX(-50%)
  margin-bottom: 45px
