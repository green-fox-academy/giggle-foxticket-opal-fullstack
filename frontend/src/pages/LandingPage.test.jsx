import React from 'react';
import { shallow } from 'enzyme';
import LandingPage from './LandingPage';

it('expect to render Landing Page component', () => {
  expect(shallow(<LandingPage />).length).toEqual(1);
});

it('should render the Landing Page component', () => {
  expect(shallow(<LandingPage />)).toMatchSnapshot();
});
