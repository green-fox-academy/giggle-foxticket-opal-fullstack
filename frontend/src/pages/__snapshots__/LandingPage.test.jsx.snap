// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  "I am the landing page!",
  <div
    className="main-features"
  >
    <div
      className="card"
    >
      <div
        className="card-image"
      >
        <img
          alt=""
          src="card-cat.jpeg"
        />
      </div>
      <div
        className="card-copy"
      >
        <h1>
          DOOM
        </h1>
        <h2>
          Video game series
        </h2>
        <p>
          Doom is a video game series and media franchise created by John Carmack, John Romero, Adrian Carmack, Kevin Cloud, and Tom Hall. The series focuses on the exploits of an unnamed space marine operating under the auspices of the Union Aerospace Corporation, who fights hordes of demons and the undead.
        </p>
      </div>
    </div>
    <div
      className="card"
    >
      <div
        className="card-image"
      >
        <img
          alt=""
          src="card-cat.jpeg"
        />
      </div>
      <div
        className="card-copy"
      >
        <h1>
          Wolfenstein
        </h1>
        <h2>
          Video game series
        </h2>
        <p>
          Wolfenstein is a series of World War II video games. It was originally created by Muse Software, before being revived by id Software. The first two games in the series, Castle Wolfenstein and Beyond Castle Wolfenstein, were developed by Muse Software and focused on stealth-based gameplay from a top-down perspective.
        </p>
      </div>
    </div>
    <div
      className="card"
    >
      <div
        className="card-image"
      >
        <img
          alt=""
          src="card-cat.jpeg"
        />
      </div>
      <div
        className="card-copy"
      >
        <h1>
          Quake
        </h1>
        <h2>
          Video game series
        </h2>
        <p>
          Quake is a series of first-person shooter video games, developed by id Software and, as of 2010, published by Bethesda Softworks. The series is composed of the eponymous game from 1996 and its nonlinear, standalone sequels which vary in setting and plot.
        </p>
      </div>
    </div>
  </div>,
]
`;
